app/models/questioning.rb

```diff
@@ -17,7 +17,7 @@ class Questioning < FormItem
 
   scope :visible, -> { where(hidden: false) }
 
-  replicable child_assocs: [:question, :condition], backward_assocs: :form,
+  replicable child_assocs: [:question, :display_conditions], backward_assocs: :form,
```

Comment:
Is there a way to document this as a place for potential complexity the next time this gets updated? Seems like :display_conditions just got more complex.

Reply:
I think it will be captured by the replication for skip logic issue. And ultimately this old replication system will be subsumed by the new, so there's that...